### DISCUSSION TITLE
Help performance

### DISCUSSION THREAD

**Discussion by rserafim** [prisma/prisma#16104] [Q&A]
Can you help me?

I need to do 2 searches in one!

- QUERY A = fetch a record by ID
- QUERY B = then search for other records similar to the ID of QUERY A

This way what I'm doing is taking almost 2 seconds, a long time.

If I just do QUERY A - the return is only 200ms.

My table has a total of 2 million records.

Is there another more performant way to do it?

![image](https://user-images.githubusercontent.com/48859044/199522069-57fa7f23-7919-4e17-9600-45005a2c858c.png)

**Comment by pantharshit00** [comment]
This looks related to https://github.com/prisma/prisma/issues/13865. Can you please add your case in that issue? Also, if possible, please share a reproduction along with a model so that we can try it out ourselves.

**Reply by pantharshit00** [reply]
Hello @rserafim 

Are you still able to reproduce this? Let me know if this is still relevant.

**Reply by rserafim** [reply]
I'm still having the same problem.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
